src/network.zig

```diff
@@ -587,29 +595,18 @@ pub const ConnectionManager = struct { // MARK: ConnectionManager
 				}
 			}
 
-			// Send a keep-alive packet roughly every 100 ms:
-			if(curTime -% lastTime > 100_000_000) {
+			// Send packets roughly every 1 ms:
+			if(curTime -% lastTime > 1000) {
```

Comment:
Magic number. General lack of unit in curTime, lastTime etc. is just a bit off putting, maybe we could alias `const Milliseconds = usize` or whatever and this way hint what is the unit.  It is only 20 lines to figure out where did we get this value from, but still.